docs/dev/shop-setup/GitHub_Free.md

```diff
@@ -0,0 +1,49 @@
+# GitHub Free Shop Setup:
+
+### Setting up the organization and user permissions:
+
+![GitHub Free Permissions Diagram](https://raw.githubusercontent.com/LibreFoodPantry/ProjectTemplate/shop_setup_documentation/docs/dev/shop-setup/diagrams/GitHub_Free_Setup_Users_And_Permissions.png)
+
+
+#### Shop Manager:
+
+1. Create a new GitHub organization for the shop
+
+2. Add the shop developers to the organization with member permissions
+
+### Setting up the shop project board:
+
+![GitHub Shop Board Diagram](https://raw.githubusercontent.com/LibreFoodPantry/ProjectTemplate/shop_setup_documentation/docs/dev/shop-setup/diagrams/GitHub_Setup_Shop_Board.png)
+
+
+#### Shop Manager:
+
+1. Create a new shop-level project board
+
+2. Add the appropriate columns to the shop board by clicking "Add column" in this order (in order on the board from left to right):
+
+    1. Backlog
+    2. In Sprint
+    3. Needs Review
+    4. Needs Merge
+    5. Done
+
+### Setting up the repository:
+
+![GitHub Organizations and Repositories Diagram](https://raw.githubusercontent.com/LibreFoodPantry/ProjectTemplate/shop_setup_documentation/docs/dev/shop-setup/diagrams/GitHub_Free_Organizations_And_Repositories.png)
+
+#### Shop Manager:
+
+1. Fork the repository you will be working on from the LibreFoodPantry organization to the shop 
+
+2. Add the shop developers to the forked repository as contributors with write permissions
+
```

Comment:
I'm moving all of the permissions stuff down to this and breaking it into a new section since on GitHub you have to add the shop developers to both the organization and then the individual repository in two separate steps.